package.json

```diff
@@ -102,6 +102,8 @@
     "openpgp": "~5.9.0",
     "path": "~0.12.7",
     "qs": "~6.11.2",
+    "splitargs": "~0.0.7",
```

Comment:
Lets move both of these to devDependencies as they are only required for UI (devDep are for UI that we are building)

Reply:
moved to devDependencies